app.js

```diff
@@ -58,7 +54,6 @@ function sumAndMultiply(a,b,c){ //eslint-disable-line
   var sumString = a + ' and ' + b + ' and ' + c + ' sum to ' + totalSum[0] + '.';
   var multiplyString = 'The product of ' + a + ' and ' + b + ' and ' + c + ' is ' + totalMultiply[0] + '.';
   var totalArray = [totalSum[0], totalMultiply[0], sumString, multiplyString];
-  console.log(totalArray);
   return totalArray;
 }
```

Comment:
Several naming issues in this function:

"result" would be better named "sumAB" as this is a sum of A and B
"totalSum" might be named "sumABC"
"multipleResult" might be better named "multiplyAB"
"totalMultiply" would be better named "multiplyABC"
"totalArray" might be better named "returnArray" or "outputArray"